settings.gradle

```diff
@@ -142,7 +142,7 @@ rootProject.name = 'airbyte'
 def cdkPath = rootDir.toPath().resolve('airbyte-cdk/java/airbyte-cdk')
 if (cdkPath.toFile().exists()) {
     cdkPath.eachDir { dir ->
-        def buildFiles = file(dir).list { file, name -> name == "build.gradle" }
+        def buildFiles = file(dir).list { file, name -> name.matches("build\\.gradle(?:\\.kts)?") }
```

Comment:
nit: why not just `name == "build.gradle" or name == "build.gradle.kts"`?

Reply:
@edgao Just for brevity.  I started with that but it makes that line very long.